addons/website_blog/controllers/main.py

```diff
@@ -31,10 +31,10 @@ def tags_list(self, tag_ids, current_tag):
 
     def nav_list(self, blog=None):
         dom = blog and [('blog_id', '=', blog.id)] or []
-        if not request.env.user.has_group('website.group_website_designer'):
-            dom += [('post_date', '<=', fields.Datetime.now())]
+        # if not request.env.user.has_group('website.group_website_designer'):
+        dom += [('website_published', '=', True)]
```

Comment:
There's a leftover of some test here.
It's unclear to me what `nav_list()` does, but either you should remove the commented condition, or you should uncomment it and add the indent back.

Reply:
Since we group by `published_date:month`, the website_published must be true. There are cases when publish_date is null and it crashes the `for` loop below. So yep it was a leftover/slag lying around... again.